tests/queries/0_stateless/03534_skip_index_bug_in_read_chain.sql

```diff
@@ -0,0 +1,31 @@
+-- Tags: no-parallel-replicas
+-- no-parallel-replicas: use_skip_indexes_on_data_read is not supported with parallel replicas.
```

Comment:
we need to check this because it has changed. IIRC `use_skip_indexes_on_data_read` now works with parallel replicas